src/csharp/Grpc.Tools/Grpc.Tools.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Grpc.Tools</id>
+    <title>gRPC C# Tools</title>
+    <summary>Tools for C# implementation of gRPC - an RPC library and framework</summary>
+    <description>Precompiled protobuf compiler and gRPC protobuf compiler plugin for generating gRPC client/server C# code. Binaries are available for Windows, Linux and MacOS.</description>
+    <version>$version$</version>
+    <authors>Google Inc.</authors>
+    <owners>grpc-packages</owners>
+    <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/grpc/grpc</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <releaseNotes>Release $version$</releaseNotes>
+    <copyright>Copyright 2015, Google Inc.</copyright>
+    <tags>gRPC RPC Protocol HTTP/2</tags>
+	<dependencies>
+	<dependency id="Grpc" version="$version$" />
+	</dependencies>
+  </metadata>
+  <files>
+    <file src="build\Grpc.Tools.targets" target="build\Grpc.Tools.targets" />
```

Comment:
I don't think it's any more desirable for it to be a solution level package than a project level package. It just happens that currently it is a solution level package and changing that require a manual intervention from everyone who has already installed the package at the solution level. Depending on the circumstances tat might be fine, I just wanted to point that out.